client/signup/steps/import/types.ts

```diff
@@ -31,3 +31,22 @@ export type FeatureName =
 	| 'plugins';
 
 export type FeatureList = { [ key in FeatureName ]: string };
+
+// List of supported importer platforms (most important)
+export type ImporterMainPlatform =
+	| 'blogger'
+	| 'medium'
+	| 'squarespace'
+	| 'wordpress'
+	| 'wix'
+	| ImporterPlatformOther;
```

Comment:
Do we need this?

Reply:
I separated the platforms cause in some functions only the most "important" are accepted and in other ones no. Obviously, we can have a single type. In the future, all platforms will have their own flow and so they will be the same.